code/modules/crew_objectives/medical_objectives.dm

```diff
@@ -1,14 +1,13 @@
 /*				MEDICAL OBJECTIVES				*/
 
 /datum/objective/crew/morgue //Ported from old Hippie
-	explanation_text = "Ensure there are no corpses on the station outside of the morgue when the shift ends."
+	explanation_text = "Ensure the Medbay has been cleaned of any corpses when the shift ends."
 	jobs = "chiefmedicalofficer,geneticist,medicaldoctor"
 
 /datum/objective/crew/morgue/check_completion()
 	for(var/mob/living/carbon/human/H in GLOB.mob_list)
```

Comment:
Change `GLOB.mob_list` to GLOB.mob_living_list`. Living refers to the typepath, not their health.

Reply:
Original code. However, can do. I presume that means it no longer searches all mobs.